### PR TITLE
Update convert.py

### PR DESCRIPTION
**Pull request by devashishdas**
# Fixed for Verbose: This part is not getting read from the __main__ function and gives a NameError
        # NameError: name 'h' is not defined
Line number 212

### CODE REVIEW

File: intermol/convert.py [hunk 1]
```diff
@@ -210,7 +210,16 @@ def main(args=None):
...
+        # Fixed for Verbose: This part is not getting read from the __main__ function and gives a NameError
+        # NameError: name 'h' is not defined
+        
+        logger.setLevel(logging.DEBUG)
+        h = logging.StreamHandler()
+        h.setLevel(logging.INFO)  # Ignores DEBUG level for now.
+        f = logging.Formatter("%(levelname)s %(asctime)s %(message)s",
+                          "%Y-%m-%d %H:%M:%S")
+        h.setFormatter(f)
+        logger.addHandler(h)
```
Comment (ctk3b) [inline intermol/convert.py:222]: The verbosity flag should really only adjust the verbosity. What do you think about simply setting the existing logger's level to DEBUG?

```suggestion
        logger.setLevel(logging.DEBUG)
```

**Review by ctk3b — COMMENTED** [review]
Nice find and thank you for the contribution! I have a suggestion to keep this change as simple as possible. Let me know what you think

**Comment by devashishdas** [comment]
Sounds Perfect!

**Comment by ctk3b** [comment]
Thanks @devashishdas !

**Pull request merged**